packages/w/wol/package.yml

```diff
@@ -12,7 +13,7 @@ description: |
     provides SecureON which extends wake-on-lan with a password. This feature
     is also provided by wol.
 setup      : |
-    %configure
+    ./configure $(sed 's|--runstatedir=/run||g' <<< "%CONFOPTS%")
```

Comment:
My apologies if this would be obvious to someone else, but why was this change made?

Reply:
Old configure scripts don't support runstatedir which is part of our default confopts

Reply:
This one ^. `%configure` and `%reconfigure` doesn't work. So i just followed this trick [here](https://github.com/getsolus/help-center-docs/issues/411)